app/controllers/miq_policy_controller/conditions.rb

```diff
@@ -157,7 +157,7 @@ def condition_build_edit_screen
 
     if params[:copy]  # If copying, create a new condition based on the original
       c = Condition.find(params[:id])
-      @condition = Condition.new(c.attributes.merge(:name => nil))
+      @condition = Condition.new(c.attributes.merge(:name => nil, :id => nil))
```

Comment:
Alternative solution: `@condition = c.dup.tap { |cond| cond.name = nil }`. `#dup` `nil`-ifies the id automatically.